README.md

```diff
@@ -41,15 +41,27 @@ A REST API to execute [teuthology commands](https://docs.ceph.com/projects/teuth
     Add following things in `teuthology_api` container:
     ```
     teuthology_api:
-        environment: 
+        environment:
             DEPLOYMENT: development
-        volumes: 
+        volumes:
             - ../../../teuthology-api:/teuthology_api/:rw
     ```
-    `DEPLOYMENT: development` would run the server in `--reload` mode (server would restart when changes are made in `/src` dir) and `volumes` would mount host directory to docker's directory (local changes would reflect in docker container). 
+    `DEPLOYMENT: development` would run the server in `--reload` mode (server would restart when changes are made in `/src` dir) and `volumes` would mount host directory to docker's directory (local changes would reflect in docker container).
 
 5. Follow teuthology development setup instructions from [here](https://github.com/ceph/teuthology/tree/main/docs/docker-compose).
 
+### Option 2: Non-containerized with venv and pip
+
+1. Clone [teuthology-api](https://github.com/VallariAg/teuthology-api) and `cd` into it.
+
+2. Create a virtualenv: `python3 -m venv`
```

Comment:
```suggestion
2. Create a virtualenv: `python3 -m venv venv`
```

Also in step 1 of both options, we can use https://github.com/ceph/teuthology-api now instead of vallariag/teuthology-api

Reply:
good catch on the venv omission! I also corrected the repo URLs.